hazelcast/proxy/map.py

```diff
@@ -245,6 +564,17 @@ def put_if_absent(self, key, value, ttl=-1):
         return self._put_if_absent_internal(key_data, value_data, ttl)
 
     def put_transient(self, key, value, ttl=-1):
+        """
+        Same as put(TKey, TValue, Ttl) but server side defined MapStore will not be called.
```

Comment:
Same as put(TKey, TValue, Ttl), but MapStore defined at the server side will not be called.